### DISCUSSION TITLE
How to customize single line code blocks in superfences

### DISCUSSION THREAD

**Discussion by CrossDark** [facelessuser/pymdown-extensions#2527] [Q&A] [answered]
I‘m trying to  define a custom fence matching `{Special Block}` and render like `<span class="block">Special Block</span>`.
But the base fence pattern `*` seemed unable to match single line code blocks, and I didn't find anything in the [document](https://facelessuser.github.io/pymdown-extensions/extensions/superfences/#custom-fences) that mentioned ways dealing with that

How can I solve that, or it's not supported

**Comment by facelessuser** [comment]
@CrossDark I'm not sure what you are asking. I'm assuming you are asking about inline code blocks? SuperFences is for block, not inline code blocks.

InlineHilite is used for inline code blocks and does allow for [creating custom inline blocks](https://facelessuser.github.io/pymdown-extensions/extensions/inlinehilite/#custom-inline-blocks). It allows for you to control the output, but it does not allow you to change the input to use braces `{}` instead of backticks `` `#!custom ` ``.

**Reply by CrossDark** [reply, marked as answer]
Thanks.Now I've switched to InlineHilite
